internal/pkg/config/flags_shared.go

```diff
@@ -19,6 +19,8 @@ const (
 	IsSudo           = "is_sudo"
 	Interpreter      = "interpreter"
 	IsFullOutput     = "full-command-response"
+	WriteExecLog     = "writeexeclog"
```

Comment:
Should be `write-execlog`. All the other flags have verb and noun separated by a dash.

Reply:
ok. done.